rdp-server/bandwidth_mgmt.cpp

```diff
@@ -248,43 +268,47 @@ BOOL ogon_bwmgmt_client_detect_rtt(ogon_connection *conn) {
 	}
 	bwmgmt->autodetect_rtt_starttime = starttime;
 	bwmgmt->autodetect_rtt_sent = TRUE;
-	peer->autodetect->RTTMeasureRequest(peer->autodetect->context,
-										bwmgmt->autodetect_rtt_seq);
+	peer->context->autodetect->RTTMeasureRequest(peer->context->autodetect,
+			RDP_TRANSPORT_TCP, bwmgmt->autodetect_rtt_seq);
 
 	return TRUE;
 }
 
-BOOL ogon_bwmgmt_client_rtt_measure_response(rdpContext *context, UINT16 sequenceNumber) {
+BOOL ogon_bwmgmt_client_rtt_measure_response(rdpAutoDetect *autodetect,
+		RDP_TRANSPORT_TYPE transport, UINT16 sequenceNumber) {
 	OGON_UNUSED(sequenceNumber);
-	ogon_connection *connection = (ogon_connection*) context;
+	ogon_connection *connection = (ogon_connection *)autodetect->context;
 	ogon_front_connection *frontend = &connection->front;
 	freerdp_peer *peer = connection->context.peer;
 	ogon_bandwidth_mgmt *bwmgmt = &connection->front.bandwidthMgmt;
 	UINT32 frameack;
 
 	if (frontend->frameAcknowledge) {
 		/* Calculate frameack based on RTT*/
-		frameack = MIN(
-			MAX( peer->autodetect->netCharBaseRTT * connection->fps / 1000, 2 ),
-			(unsigned int) connection->fps );
+		frameack = MIN(MAX(peer->context->autodetect->netCharBaseRTT *
```

Comment:
previous code style was easier to read (perhaps that shall be splitted in 2 operations)